### PR TITLE
Ecommerce suborder extension webhook doc

### PR DESCRIPTION
**Pull request by cvrac**

### CODE REVIEW

File: source/localizable/smart_cart/webhooks.html.md.erb [hunk 1]
```diff
@@ -79,7 +79,10 @@ A **new order** webhook request is triggered upon the placement of a new order.
 ### Order updates
 
 An **order update** event is triggered upon the update of an order. As of right now, an update
-event notification is sent on order **cancellation**.
+event notification is sent for the following cases:
+
+* Order **cancellation**, triggered if an order has been cancelled by either the user or our support.
+* Order expiration time **extension**,  triggered if an order's expiration time has been extended.
```
Comment (astathopoulos) [inline source/localizable/smart_cart/webhooks.html.md.erb]: It's not expiration time, it's the latest time the merchant should dispatch the order.

Comment (astathopoulos) [reply source/localizable/smart_cart/webhooks.html.md.erb]: Well, extension action changes both timestamps so we should rephrase that in a more generic way like "the available time to handle the order is extended" or smth

Comment (cvrac) [reply source/localizable/smart_cart/webhooks.html.md.erb]: How about the following:
* Order **extension**,  triggered if the available time to handle the order is extended.

Comment (nikostsaganos) [reply source/localizable/smart_cart/webhooks.html.md.erb]: Another option would be

```
- When an order is **cancelled** (either the user or our support team).
- When an order is **extended** (allowing more time to be handled).
```

File: source/localizable/smart_cart/webhooks.html.md.erb [hunk 2]
```diff
@@ -94,7 +97,9 @@ Name           | Type   | Value | Description
 
 Name           | Type   | Value | Description
 :------------: | :----: | :---: | :-----------:
-`state` | Object | `{ "old": "open", "new": "cancelled" }` | The state changed values
+`state` | Object | `{ "old": "open", "new": "cancelled" }` | The state changed values (optional)
+`expires_at` | Object | `{ "old": "YYYY-MM-DDTHH:MM:SS+HH:M", "new": "YYYY-MM-DDTHH:MM:SS+HH:M" }` | The order's current expiration time changes (optional)
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/webhooks.html.md.erb]: Should we add actual sample values here? (we're in the "values" column, not the type)

File: source/localizable/smart_cart/webhooks.html.md.erb (hunk 1 above)
Comment (cvrac) [reply source/localizable/smart_cart/webhooks.html.md.erb]: I agree.

File: source/localizable/smart_cart/webhooks.html.md.erb (hunk 2 above)
Comment (cvrac) [reply source/localizable/smart_cart/webhooks.html.md.erb]: Added specific values for this one.

**Pull request merged**